proto/frequenz/api/dispatch/dispatch.proto

```diff
@@ -191,9 +188,12 @@ message DispatchUpdateRequest {
   // The creation time
   optional google.protobuf.Timestamp create_time = 7;
 
-  // The dispatch status
-  optional DispatchStatus status = 8;
+  // The "active" status
+  optional bool status = 8;
```

Comment:
I'd suggest being more consistent here, and go with `active` (or `is_active`), instead of `status`.

Reply:
Oops, missed this one.